Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Build stage
+FROM python:3.8 AS builder
+
+ENV PYTHONUNBUFFERED=1
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Copy the requirements file first to leverage Docker cache
+COPY requirements.txt /app/
+
+# Install the Python dependencies
+RUN pip install --no-cache-dir -r requirements.txt
+
+# Copy the rest of the application code into the container
+COPY . /app
+
+# Run stage
+FROM python:3.8-slim
+
+ENV PYTHONUNBUFFERED=1
+
+# Set the working directory inside the container
+WORKDIR /app
+
+# Copy the installed dependencies from the build stage
+COPY --from=builder /usr/local/lib/python3.8/site-packages /usr/local/lib/python3.8/site-packages
+COPY --from=builder /usr/local/bin /usr/local/bin
+
+# Copy the application code from the build stage
+COPY --from=builder /app /app
+
+# Expose port 8080 to the host
+EXPOSE 8080
+
+# Set the command to run the Django development server
+ENTRYPOINT ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8080"]
```

Comment:
According to the task description migration should be done by `RUN` command in the process of building the image 
![image](https://github.com/mate-academy/devops_todolist/assets/87720583/fb466472-3a16-4867-b89e-6197ece700d4)